ps3t_game_list.php

```diff
@@ -91,94 +77,77 @@
             echo '<B>XBOX ONE</B><BR>'; }
         else if ($gametype[$i] == 'app') {
             echo '<B>APPLICATIONS</B><BR>'; }
-        else 	{
+        else {
             echo '<B>' . str_replace('%20',' ',strtoupper($gametype[$i])) . '</B><BR>'; }
         
-        $nextpage = 1;
+        // Set number of pages dynamically
+        $init_html = file_get_html('http://www.' . $siteurl . '/browsegames/' . $gametype[$i] . '/' . $gamealpha . '/');
         
-        for ($j = 1; $j <= 9; $j++) {
-            
-            $k = $j+1;
+        if ($gamesite == 'ps3t') {
+            $table_object = $init_html->find('div.bl_la_main div.divtext table', 1);
+            $page_object = $table_object->find('td', 0);
+            $page_num = explode(" ", $page_object)[4]; }

```

Comment:
```suggestion
            $page_num = substr(explode(" ", $page_object)[4], 0, -3); }

```

Reply:
Tested some more... for PST you need to trim the last 3 characters for the pagenum, the way you had it returns:

"5"<br

Which works but let's make it cleaner.